### DISCUSSION TITLE
Can't verify my status as teacher

### DISCUSSION THREAD

**Discussion by lupaulus** [community/community#78466] [GitHub Education] [answered]
### Hi everyone,

I can't verify my status as Teacher, I have documents from my University with the contract. 
But the robot processing it, refuse it.

**Comment by HiWebDev0026** [comment, marked as answer]
Hi There!
 Ensure that the documents you're submitting are clear, complete, and meet the requirements for verification. It's essential to double-check for any discrepancies or missing information.

 Reach out to the system or organization handling the verification process to understand why your documents are being refused. Request specific feedback on what requirements are not being met and if there are any alternative documents that might be accepted.

 Explore if there are additional documents or information you can provide to strengthen your verification attempt. This could include letters of recommendation, additional certifications, or any other relevant paperwork that supports your status as a teacher.

 If possible, try to connect with a human representative who can provide more personalized guidance through the verification process. They may be able to offer specific insights or alternative solutions to help you resolve the issue.

By addressing these steps, you will be better equipped to understand the reasons for the refusal and take the necessary actions to successfully verify your status as a teacher.
